### PR TITLE
Update Keepalived unicast config validation

### PR DESCRIPTION
**Pull request by yboaron**
In the case of unicast, Keepalived configuration on masters/workers can only
be updated if the node appears in the backends list.

### CODE REVIEW

**Comment by yboaron** [comment]
/retitle Bug 1913705: Update Keepalived unicast config validation

**Review by cybertron — CHANGES_REQUESTED** [review]
Can we just modify the keepalived signal handler to ignore reload if keepalived was previously stopped? IIUC, that would also fix the problem here and it would be a lot simpler than adding more thresholds.

File: pkg/monitor/dynkeepalived.go [hunk 1]
```diff
@@ -255,6 +289,8 @@ func KeepalivedWatch(kubeconfigPath, clusterConfigPath, templatePath, cfgPath st
 	var appliedConfig, curConfig, prevConfig *config.Node
 	var configChangeCtr uint8 = 0
 
+	log.Info("Test version Test Version Latest ! !!!!!!!!!")
```
[the remark this answers is not in the document]
Comment (yboaron) [reply pkg/monitor/dynkeepalived.go:292]: Marius uses the ci BOT to create a release with this PR for testing , we wanted to make sure he was using the correct baremetal-runtimecfg image ...

I'll delete this line

**Comment by yboaron** [comment]
> Can we just modify the keepalived signal handler to ignore reload if keepalived was previously stopped? IIUC, that would also fix the problem here and it would be a lot simpler than adding more thresholds.

Well, I added this change for two reasons:
A.  Make sure that sending multiple reload messages doesn't cause Keepalived or Keepalived-monitor  hang on for ~ 20 minutes. 
B.  Check if adding threshold also for start helps.

It seems that this change didn't solve the hang on issue , so I think I'll delete this change from PR.

**Pull request closed without merging**